src/test/kotlin/com/petqua/presentation/payment/PaymentControllerSteps.kt

```diff
@@ -0,0 +1,52 @@
+package com.petqua.presentation.payment
+
+import io.restassured.module.kotlin.extensions.Extract
+import io.restassured.module.kotlin.extensions.Given
+import io.restassured.module.kotlin.extensions.Then
+import io.restassured.module.kotlin.extensions.When
+import io.restassured.response.Response
+
+fun requestSucceedPayment(
+    accessToken: String,
+    succeedPaymentRequest: SucceedPaymentRequest,
+): Response {
+    return Given {
+        log().all()
+        auth().preemptive().oauth2(accessToken)
+        params(
+            "paymentType", succeedPaymentRequest.paymentType,
+            "orderId", succeedPaymentRequest.orderId,
+            "paymentKey", succeedPaymentRequest.paymentKey,
+            "amount", succeedPaymentRequest.amount
+        )
+    } When {
+        post("/orders/payment/success")
+    } Then {
+        log().all()
+    } Extract {
+        response()
+    }
+}
+
+fun requestFailPayment(
+    accessToken: String,
+    failPaymentRequest: FailPaymentRequest,
+): Response {
+    val paramMap = mutableMapOf<String, Any?>().apply {
+        put("code", failPaymentRequest.code)
+        put("message", failPaymentRequest.message)
+        put("orderId", failPaymentRequest.orderId)
+    }.filterValues { it != null }
```

Comment:
nullable할 필드 때문에 이렇게 사용하셨군요!
아래 처럼 바로 선언 해도 괜찮을 것 같아요! 어차피 새로운 map으로 반환해서 mutable로 안해도 될 것 같습니다.
```suggestion
    val paramMap = mapOf(
            "code" to failPaymentRequest.code,
            "message" to failPaymentRequest.message,
            "orderId" to failPaymentRequest.orderId,
        ).filterValues { it != null }
```